bazel/repositories.bzl

```diff
@@ -1,7 +1,7 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-ENVOY_COMMIT = "f1a3fd1c549e86a02aa222784bcef9bf63dca19d"
-ENVOY_SHA = "092e7243de4cd49e1ac9de611996a9dcc9fa2f398ef5e3f92fc90269abfcbfab"
+ENVOY_COMMIT = "84b66829bdda86c719047ccc3c0a4fc2f2e2c328"  # May 23, 2022
```

Comment:
(nit) Please remove the date comment after the commit. We have recently removed it here and from the documentation because it requires manual updates and tends to be error prone.

Reply:
ahh done